pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -653,34 +653,35 @@ func (r *ReconcileComplianceRemediation) verifyAndCompleteKC(obj *unstructured.U
 	// we need to patch the remediation if there is already a CustomKC present
 	if hasCustomKC {
 
-		kubletMC := &mcfgv1.MachineConfig{}
+		kubeletMC := &mcfgv1.MachineConfig{}
 		kMCKey := types.NamespacedName{Name: kubeletMCName}
 
-		if err := r.client.Get(context.TODO(), kMCKey, kubletMC); err != nil {
+		if err := r.client.Get(context.TODO(), kMCKey, kubeletMC); err != nil {
 			return fmt.Errorf("couldn't get current generated KubeletConfig MC: %w", err)
 		}
-		// WE need to get name of original kubelet config that used to generate this kubeletconfig machine config
-		if len(kubletMC.GetOwnerReferences()) == 0 {
-			return fmt.Errorf("the generated kubelet machine config does not have a owner")
```

Comment:
That's a good question, I removed this error return because, if the mc does not have an owner, it will assume `99-` is not generated by kubeletconfig, and create a kubeletconfig object instead.